pkgs/by-name/ov/ovftool/package.nix

```diff
@@ -85,6 +85,15 @@ let
   };
 
   ovftoolSystem = ovftoolSystems.${stdenv.system} or (throw "unsupported system ${stdenv.system}");
+
+  # Regrettably, we need to compile this version or else ovftool complains about unknown symbols.
+  ovftool-xercesc = xercesc.overrideAttrs (prev: rec {
```

Comment:
lame

Reply:
... yep. The xerces-c symbols have versions in them.